import java.util.*;

abstract class Staff {
    String name;

    Staff(String name) {
        this.name = name;
    }

    abstract double getPay();
}

class FullTime extends Staff {
    double salary;

    FullTime(String name, double salary) {
        super(name);
        this.salary = salary;
    }

    double getPay() {
        return salary;
    }
}

class Hourly extends Staff {
    double hours, rate;

    Hourly(String name, double hours, double rate) {
        super(name);
        this.hours = hours;
        this.rate = rate;
    }

    double getPay() {
        if (hours <= 40)
            return hours * rate;

        return 40 * rate + (hours - 40) * rate * 1.5;
    }
}

class Intern extends Staff {
    double stipend;

    Intern(String name, double stipend) {
        super(name);
        this.stipend = stipend;
    }

    double getPay() {
        return stipend;
    }
}

class WeeklyPay {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        double total = 0;

        for (int i = 0; i < n; i++) {
            String type = sc.next();
            String name = sc.next();

            Staff s;

            if (type.equals("FULLTIME")) {
                s = new FullTime(name, sc.nextDouble());
            } else if (type.equals("HOURLY")) {
                s = new Hourly(name, sc.nextDouble(), sc.nextDouble());
            } else {
                s = new Intern(name, sc.nextDouble());
            }

            double pay = s.getPay();

            System.out.printf("%s: %.2f%n", name, pay);

            total += pay;
        }

        System.out.printf("Total Payroll: %.2f%n", total);
    }
}
